tests/test_unit.py

```diff
@@ -62,6 +62,37 @@ def test_retry(test_consumer):
     )
 
 
+@pytest.mark.asyncio
+def test_produce_exhange_name(test_producer):
+    """Test that providing an exhange name when sending works."""
```

Comment:
Test that providing an `exchange` name when sending`,` works.
OR
Test that providing an `exchange` name `works when sending.`